hpat/tests/test_io.py

```diff
@@ -1,4 +1,5 @@
 import unittest
+import sys
```

Comment:
Instead of sys.platform use platform.system(). This value is more predictable.

Reply:
@dmitrii-zagornyi Good catch!
"The output of sys.platform and os.name are determined at compile time. platform.system() determines the system type at run time."
https://stackoverflow.com/a/11674977/4428424

Reply:
done